react/README.md

```diff
@@ -1,757 +1,776 @@
 # Airbnb React/JSX Style Guide
 
-*A mostly reasonable approach to React and JSX*
+_A mostly reasonable approach to React and JSX_
```

Comment:
please do not run prettier on projects that don't have it configured. it makes things uglier.

Reply:
Hi, This is the first contribute for me.
What this mean : (This diff is impossible to review with unrelated formatting changes), and what is your advice?
Thanks.

Reply:
I'd revert the file, disable Prettier in your editor, and then make your change, commit it, and force push to the branch.

Reply:
> please do not run prettier on projects that don't have it configured. it makes things uglier.
Got it, thanks.

Reply:
> I'd revert the file, disable Prettier in your editor, and then make your change, commit it, and force push to the branch.

Do I have to pull request again after the force push to branch?

Reply:
No, definitely not. This PR will update from the branch.

Reply:
> No, definitely not. This PR will update from the branch.

Thanks, I appreciate your help.